src/0x.ts

```diff
@@ -293,4 +281,28 @@ export class ZeroEx {
         }
         return proxyAuthorizedExchangeContractAddresses;
     }
+    private parseSignatureHexAsVRS(signatureHex: string): ECSignature {
+        const signatureBuffer = ethUtil.toBuffer(signatureHex);
+        let v = signatureBuffer[0];
+        if (v < 27) {
+            v += 27;
+        }
+        const r = signatureBuffer.slice(1, 33);
+        const s = signatureBuffer.slice(33, 65);
+        const ecSignature: ECSignature = {
```

Comment:
I don't consider this a serious form of code duplication. It doesn't include any logic, only instantiating an instance of the return type. It's also only in two places so it's fine for now.